runtime/queries/rust/highlights.scm

```diff
@@ -106,12 +106,17 @@
   (string_literal)
   (raw_string_literal)
 ] @string
-(outer_doc_comment_marker "/" @comment)
-(inner_doc_comment_marker "!" @comment)
+
 [
   (line_comment)
   (block_comment)
 ] @comment
+(line_comment
+  (outer_doc_comment_marker "/" @comment.block.documentation)
+  (doc_comment)) @comment.block.documentation
+(line_comment
```

Comment:
For these we should tag as `comment.line.documentation` - Rust also has block doc comments ([reference](https://doc.rust-lang.org/reference/comments.html)) as

```rust
/**  - Outer block doc (exactly) 2 asterisks */
```

which we could tag as well